getting-started/iterate-new-app-intellij.hbs.md

```diff
@@ -0,0 +1,141 @@
+
+You are now ready to iterate on your application.
+
+## <a id="apply-your-app"></a>Apply your application to the cluster
+
+Apply your application to the cluster to get your application running.
+
+
+1. In the Project tab of IntelliJ, right-click any file under the application name `tanzu-java-web-app` and select `Tanzu` > `Apply Workload`.
+1. In the dialog box enter your **Source Image**, **Local Path**, and optionally a **Namespace**.
+
+    1. In the **Source Image** field, provide the destination image repository to publish an image containing your workload source code. The Source Image value tells the Tanzu Developer Tools extension where to publish the container image with your uncompiled source code, and what to name that image. The image must be published to a container registry where you have write (push) access.
+    For example, `gcr.io/myteam/tanzu-java-web-app-source`.
+
+    >**Note** Consult the documentation for the registry you're using to determine which steps are necessary to authenticate and gain push access.
+
+    >For example, if you use docker consult [docker's docs](https://docs.docker.com/engine/reference/commandline/login/), if you use Harbor consult [Harbor's docs](https://goharbor.io/docs/1.10/working-with-projects/working-with-images/pulling-pushing-images/), etc.
+
+    2. In the **Local Path** field, provide the path to the directory containing the Tanzu Java Web App. The current directory is the default. The Local Path value tells the Tanzu Developer Tools extension which directory on your local file system to bring into the source image container image.
+    For example, `.` uses the working directory, or you can specify a full file path.
+    3. In the **Namespace** field, optionally provide the namespace that the workload should be associated with on the cluster. If you followed the steps to [Prepare your IDE to iterate on your application](/iterate-new-app-intellij.hbs.md#prepare-to-iterate) you do not need to enter a namespace and IntelliJ will use the namespace you associated with your context.
+
+The Apply Workload command will run, which opens a terminal and shows you the output of the Workload Apply. You can also monitor your application as it's being deployed to the cluster on the `Tanzu Panel`. The `Tanzu Panel` shows the workloads in the namespace associated with your current Kubernetes context on the left side, and the details of the Kubernetes resources for the workloads running in the namespace associated with your current Kubernetes context in the center. The Apply Workload command can take a few minutes to deploy your application onto the cluster.
+
+
+## <a id="live-update-your-app"></a>Live update your application
+
+Deploy the application to view it updating live on the cluster to see how code changes behave on a production cluster.
+
+Follow the following steps to live update your application:
+
+1. Create a Run Configuration.
```

Comment:
First time only

Reply:
Good point. This content is in the getting started guide, so I think we're safe to assume someone is doing all of this for the first time. If we use any of this in our general help docs we should keep this in mind.